mobile/src/main/java/org/openhab/habdroid/core/OpenHABVoiceService.java

```diff
@@ -203,7 +224,7 @@ private void showToast(final String message) {
         new Handler(Looper.getMainLooper()).post(new Runnable() {
             @Override
             public void run() {
-                Toast.makeText(getApplicationContext(), message, Toast.LENGTH_LONG).show();
+                showMessageToUser(message, Constants.MESSAGES.SNACKBAR, Constants.MESSAGES.LOGLEVEL.ALWAYS);
```

Comment:
That would never show anything, as MESSAGES.SNACKBAR is not handled, right?

Reply:
Right, I missed that. I change this to a toast since I want to be sure that it will be displayed